### DISCUSSION TITLE
PCL2好玩吗

### DISCUSSION THREAD

**Discussion by TCTCsx** [Meloong-Git/PCL#3150] [讨论]
PCL2好玩吗

**Comment by MoYuan-CN** [comment]
啊？

**Comment by SuoHouDaiShi** [comment]
啊？

**Comment by allMagicNB** [comment]
啊？

**Comment by NiMing561** [comment]
啊？

**Comment by TCTCsx** [comment]
说着玩的

**Comment by allMagicNB** [comment]
> 说着玩的
> 
> 从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送
> 
> 发件人: ***@***.***>
> 发送时间: 2024年1月19日 0:38
> 收件人: ***@***.***>
> 抄送: ***@***.***>; ***@***.***>
> 主题: Re: [Hex-Dragon/PCL2] PCL2好玩吗 (Discussion #3150)
> 
> 
> 啊？
> 
> ―
> Reply to this email directly, view it on GitHub<https://github.com/Hex-Dragon/PCL2/discussions/3150#discussioncomment-8171620>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BFLZQS3MRAQIMSFX5YKAI53YPFFWNAVCNFSM6AAAAABB4C6OSWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCNZRGYZDA>.
> You are receiving this because you authored the thread.Message ID: ***@***.***>
> 
> 

6

**Comment by TCTCsx** [comment]
但是怎么都跟我说 “啊？”

**Comment by SuoHouDaiShi** [comment]
> 但是怎么都跟我说 “啊？” 从 Windows 版邮件<[https://go.microsoft.com/fwlink/?LinkId=550986>发送](https://go.microsoft.com/fwlink/?LinkId=550986>%E5%8F%91%E9%80%81) 发件人: ***@***.***> 发送时间: 2024年1月19日 13:24 收件人: ***@***.***> 抄送: ***@***.***>; ***@***.***> 主题: Re: [Hex-Dragon/PCL2] PCL2好玩吗 (Discussion #3150) 说着玩的 从 Windows 版邮件[https://go.microsoft.com/fwlink/?LinkId=550986发送](https://go.microsoft.com/fwlink/?LinkId=550986%E5%8F%91%E9%80%81)……

@TCTCsx 哥你上 Github 回复不行吗，用邮箱发有很多没用的信息的……

**Comment by TCTCsx** [comment]
好的

**Reply by allMagicNB** [reply]
《好的》

**Reply by MoYuan-CN** [reply]
《好的》

**Comment by iPodtouch6557** [comment]
wothefak
